mmv1/products/biglake/Table.yaml

```diff
@@ -0,0 +1,142 @@
+    name: "updateTime"
+    description: |
+      Output only. The last modification time of the table. A timestamp in
+      RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: "2014-10-02T15:01:23Z" and
+      "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "deleteTime"
+    description: |
+      Output only. The deletion time of the table. Only set after the
+      table is deleted. A timestamp in RFC3339 UTC "Zulu" format, with
+      nanosecond resolution and up to nine fractional digits. Examples:
+      "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "expireTime"
+    description: |
+      Output only. The time when this table is considered expired. Only set
+      after the table is deleted. A timestamp in RFC3339 UTC "Zulu" format,
+      with nanosecond resolution and up to nine fractional digits. Examples:
+      "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "etag"
+    description: |
+      The checksum of a table object computed by the server based on the value
+      of other fields. It may be sent on update requests to ensure the client
+      has an up-to-date value before proceeding. It is only checked for update
+      table operations.
+    output: true
+
+  - !ruby/object:Api::Type::Enum
+    name: "type"
+    description: |
+      The database type.
+    values:
+      - :HIVE
+  - !ruby/object:Api::Type::NestedObject
+    name: "hiveOptions"
+    description: |
+      Options of a Hive table.
+    properties:
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "parameters"
+        description: |
+          Stores user supplied Hive table parameters. An object containing a
+          list of "key": value pairs.
+          Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
+      - !ruby/object:Api::Type::String
+        name: "tableType"
+        description: |
+          Hive table type. For example, MANAGED_TABLE, EXTERNAL_TABLE.
+      - !ruby/object:Api::Type::NestedObject
+        name: "storageDescriptor"
+        description: |
+          Stores physical storage information on the data.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "locationUri"
+            description: |
+              Cloud Storage folder URI where the table data is stored, starting with "gs://".
+          - !ruby/object:Api::Type::String
+            name: "inputFormat"
+            description: |
+              The fully qualified Java class name of the input format.
+          - !ruby/object:Api::Type::String
+            name: "outputFormat"
+            description: |
+              The fully qualified Java class name of the output format.
+          - !ruby/object:Api::Type::NestedObject
+            name: "serdeInfo"
```

Comment:
@nevzheng it looks like the field still needs to be removed here.

Reply:
done.